supervision/detection/core.py

```diff
@@ -324,6 +326,28 @@ def from_roboflow(cls, roboflow_result: dict, class_list: List[str]) -> Detectio
             class_id.append(class_list.index(prediction["class"]))
             confidence.append(prediction["confidence"])
 
+            if prediction.get("points"):
```

Comment:
Let's prevent adding indented logic - reverse the logical condition and use `continue`.

```python
if "points" not in prediction:
    continue
```